service/src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -187,16 +182,13 @@ private boolean deleteResourceInner(UUID workspaceId, UUID resourceId) {
    */
   public List<WsmResource> enumerateResources(
       UUID workspaceId,
-      @Nullable List<String> controlledResourceIds,
       @Nullable WsmResourceType resourceType,
       @Nullable StewardshipType stewardshipType,
       int offset,
       int limit)
       throws InterruptedException {
     return DbRetryUtils.retry(
-        () ->
-            enumerateResourcesInner(
-                workspaceId, controlledResourceIds, resourceType, stewardshipType, offset, limit));
+        () -> enumerateResourcesInner(workspaceId, resourceType, stewardshipType, offset, limit));
   }
 
   @Transactional(
```

Comment:
We still have transactional on a private method. Is there a bug for this?

Reply:
yep, that's tracked in PF-885, and I'd expect we probably pull that in within ~ a sprint. I didn't want to change just this method since it's an issue with all the DAO methods, and we might pull in a common-lib solution.

Reply:
Yes, we should switch to the common-lib version.